### PR TITLE
Add 'loading' field to images in wysiwyg editor

### LINKED ISSUE #15790: We should be able to set the 'loading' field in images with WYSIWYG editor
### Describe the Bug

Not really a bug but a missing feature / improvement. 

`` tags support a 'loading' field which defines if it should be lazy loaded or not. It's quite useful for performance in blog websites with long articles with multiple images.

### To Reproduce

Add an image with the wysiwyg editor, you can't set the loading field

### Errors Shown

_No response_

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

LTS

### What database are you using?

Postgres 13

### What browser are you using?

irrelevant

### How are you deploying Directus?

irrelevant

### PR DESCRIPTION
**Pull request by izoukhai**
## Description

Capture d’écran 2022-09-28 à 17 30 57

Add a `loading` field when uploading images with wysiwyg

Fixes #15790

## Type of Change

- [ ] Bugfix
- [X] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [X] All tests are passing locally
- [X] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated. PR:

### CODE REVIEW

**Comment by benhaynes** [comment]
Thank you! I wonder why URL wasn't capitalized per our Title Formatter. Also, I think one of those first two fields should be full with and we move the loading field up so the form feels more balanced.

**Comment by izoukhai** [comment]
> 

yep good idea, I pushed the modification :)

File: app/src/interfaces/input-rich-text-html/input-rich-text-html.vue [hunk 1]
```diff
@@ -204,6 +215,10 @@ import 'tinymce/plugins/paste/plugin';
 import 'tinymce/plugins/preview/plugin';
 import 'tinymce/plugins/table/plugin';
 import 'tinymce/icons/default';
+import VSelect from '@/components/v-select/v-select.vue';
+import VItem from '@/components/v-item.vue';
+import VSelect1 from '@/components/v-select/v-select.vue';
+import SelectDropdown from '../select-dropdown/select-dropdown.vue';
```
Comment (br41nslug) [inline app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:221]: These are not needed as `v-select` is registered globally.
```suggestion
```

File: app/src/interfaces/input-rich-text-html/input-rich-text-html.vue [hunk 2]
```diff
@@ -214,7 +229,7 @@ type CustomFormat = {
 };
 
 export default defineComponent({
-	components: { Editor },
+	components: { Editor, VSelect },
```
Comment (br41nslug) [inline app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:232]:
```suggestion
	components: { Editor },
```

File: packages/shared/src/types/settings.ts [hunk 3]
```diff
@@ -24,6 +24,7 @@ export type SettingsStorageAssetPreset = {
 	withoutEnlargement: boolean | null;
 	format: 'jpeg' | 'png' | 'webp' | 'tiff' | null;
 	transforms: any[] | null;
+	loading: string | null;
```
Comment (br41nslug) [inline packages/shared/src/types/settings.ts:27]:
```suggestion
	loading: 'lazy' | 'eager' | null;
```

**Review by br41nslug — CHANGES_REQUESTED** [review]
Thanks for your contribution 😄 lets remove [api/directus-sqlite](https://github.com/directus/directus/pull/15791/files#diff-c60ba541cc749f58d89ea2f574aaad7dcf7120d4cb8ee824c3efedd09a044bf9) however which im assuming is an accidentally committed SQLite database.

**Comment by rijkvanzanten** [comment]
Closing this for now, as the review has gone ignored

**Pull request closed without merging**